docs/user_docs/installation/upgrade/upgrade-kubeblocks-to-0.7.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Upgrade to KubeBlocks v0.7
+description: Upgrade to KubeBlocks, operation, tips and notes
+keywords: [upgrade, 0.7]
+sidebar_position: 1
+sidebar_label: Upgrade to KubeBlocks v0.7
+---
+
+# Upgrade to KubeBlocks v0.7
+
+Upgrade KubeBlocks from v0.6 to v0.7 only requires a command line.
```

Comment:
Install and ensure the kbcli version is 0.7 first.

Reply:
> Install and ensure the kbcli version is 0.7 first.

Fixed.